homeassistant/components/knx/strings.json

```diff
@@ -0,0 +1,42 @@
+{
+  "config": {
+    "step": {
+      "type": {
+        "description": "Please enter the connection type we should use for your KNX connection. \n AUTOMATIC - The integration takes care of the connectivity to your KNX Bus by performing a gateway scan. \n TUNNELING - The integration will connect to your KNX bus via tunneling. \n ROUTING - The integration will connect to your KNX bus via routing.",
```

Comment:
A lot of implementation details here. If we have automatic, why would a user ever want something else ?

Reply:
Automatic connects to the first responding knx interface via Tunnelling. There are a bunch of reasons a user could not want this, but set a specific interface to be connected to (or use routing which doesn't connect to an interface at all).